manifest.json

```diff
@@ -74,5 +74,16 @@
       "entrypoint": "languageFilterEntry",
         "type": "textarea",
          "page": "threads"
-    }
+    },
+        {
+      "name": "Kbin Easy Emoticon",
+        "author": "minnieo",
+            "version": "0.6.1",
+        "label": "Easy emoticons",
+        "desc": "Use slash commands for emoticons",
```

Comment:
You can add a link to the guide by adding `<a href="https://docs.google.com/document/d/1jMeY7GwmgwrV8mEC4jmkEKKWdKuWvjgLgYrtUS5hPYg/edit">Command guide here</a>` to the description